siliconcompiler/tools/openroad/openroad.py

```diff
@@ -196,10 +196,10 @@ def setup(chip, mode='batch'):
                             ('grt_macro_extension', '2'),
                             ('grt_allow_congestion', 'False'),
                             ('grt_allow_overflow', 'False'),
-                            ('grt_signal_min_layer', chip.get('asic', 'minlayer')),
-                            ('grt_signal_max_layer', chip.get('asic', 'maxlayer')),
-                            ('grt_clock_min_layer', chip.get('asic', 'minlayer')),
-                            ('grt_clock_max_layer', chip.get('asic', 'maxlayer')),
+                            ('grt_signal_min_layer', chip.get('constraint', 'net', '*', 'minlayer')),
+                            ('grt_signal_max_layer', chip.get('constraint', 'net', '*', 'maxlayer')),
+                            ('grt_clock_min_layer', chip.get('constraint', 'net', '*', 'minlayer')),
+                            ('grt_clock_max_layer', chip.get('constraint', 'net', '*', 'maxlayer')),
```

Comment:
we should probably check if these are set before calling them and add a requirement for them for OpenROAD.

Reply:
Thought about it some more, and I am not quite there with the global constraints yet....needs more consideration.